pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -104,6 +107,14 @@ func NewNodeController(
 	nodeMonitorGracePeriod time.Duration,
 	nodeStartupGracePeriod time.Duration,
 	nodeMonitorPeriod time.Duration) *NodeController {
+	eventBroadcaster := record.NewBroadcaster()
+	recorder := eventBroadcaster.NewRecorder(api.EventSource{Component: "controllermanager"})
+	if kubeClient != nil {
+		glog.Infof("Sending events to api server.")
+		eventBroadcaster.StartRecordingToSink(kubeClient.Events(""))
```

Comment:
IIUC the argument to kubeClient.Events() is the namespace? I guess "" is fine but I was just curious why it wasn't "default" or something like that.